File: JoyalAJohney/implementation/apple-and-orange.py
#!/bin/python3

import sys


s,t = input().strip().split(' ')
s,t = [int(s),int(t)]
a,b = input().strip().split(' ')
a,b = [int(a),int(b)]
m,n = input().strip().split(' ')
m,n = [int(m),int(n)]
apple = [int(apple_temp) for apple_temp in input().strip().split(' ')]
orange = [int(orange_temp) for orange_temp in input().strip().split(' ')]
apple_count,orange_count=0,0

for apples_d in apple:
    dist=a+(apples_d)
    if dist>=s and dist<=t:
        apple_count+=1

for orange_d in orange:
    dist=b+(orange_d)
    if dist>=s and dist<=t:
        orange_count+=1
        
print(apple_count)
print(orange_count)

File: AshmiFathima/AppleAndOrange.cpp

#include <iostream>


using namespace std;


int main(){
    int s;
    int t;
    cin >> s >> t;
    int a;
    int b;
    cin >> a >> b;
    int m;
    int n;
    cin >> m >> n;
    int applecount=0;
    int orangecount=0;
    int i,d;
    for(i=0;i<m;++i)
    {
        cin>>d;
        d+=a;
        if(d>=s && d<=t)
            applecount++;
    }
    for(i=0;i<n;++i)
    {
        cin>>d;
        d+=b;
        if(d>=s && d<=t)
            orangecount++;
    }
    cout<<applecount<<endl;
    cout<<orangecount<<endl;
    return 0;
}
